packages/realm/src/Realm.ts

```diff
@@ -1271,6 +1317,10 @@ export namespace Realm {
   export import ObjectClass = internal.RealmObjectConstructor;
   /** @deprecated Will be removed in v13.0.0. Please use {@link internal.PropertyTypeName} */
   export import PropertyType = internal.PropertyTypeName;
+  /** @deprecated Use the another {@link internal.ClientResetMode} than {@link internal.ClientResetMode.Manual}. */
```

Comment:
To show slightly friendlier names for users you can show just the part excluding `internal.` (e.g. as such: [previous comment](https://github.com/realm/realm-js/pull/6492#discussion_r1500544652). (Just readding it because I don't know if you saw the other reply 👍 )

Reply:
I wasn't sure why you suggested that - but now it makes perfect sense. Thanks!